base/loading.jl

```diff
@@ -66,40 +67,37 @@ function _require_from_serialized(node::Int, path_to_try::ByteString, toplevel_l
         else
             content = remotecall_fetch(node, open, readbytes, path_to_try)
         end
-        if _include_from_serialized(content)
+        restored = _include_from_serialized(content)
+        if restored != nothing
```

Comment:
Since the type of `restored` is `Any`, it's probably better to use `===` and `!==` when comparing to `nothing`.